test/apis/v1alpha1/api_validation_integration_test.go

```diff
@@ -23,104 +23,102 @@ var _ = Describe("Test networking v1alpha1 API validation", func() {
 
 	Context("Test MultiClusterService API validation - invalid cases", func() {
 		It("should deny creating API with invalid name size", func() {
-			var name = "abcdef-123456789-123456789-123456789-123456789-123456789-123456789-123456789"
 
 			// Create the API.
 			multiClusterServiceName := &v1alpha1.MultiClusterService{
 				ObjectMeta: metav1.ObjectMeta{
-					Name:      name,
-					Namespace: "testnamespace",
+					Name:      nameWithInvalidSize,
+					Namespace: testNamespace,
 				},
 				Spec: v1alpha1.MultiClusterServiceSpec{
 					ServiceImport: v1alpha1.ServiceImportRef{
 						Name: "service-import-1",
 					},
 				},
 			}
-			By(fmt.Sprintf("expecting denial of CREATE API %s", name))
+			By(fmt.Sprintf("expecting denial of CREATE API %s", nameWithInvalidSize))
 			var err = hubClient.Create(ctx, multiClusterServiceName)
 			var statusErr *k8serrors.StatusError
-			fmt.Print(err)
 			Expect(errors.As(err, &statusErr)).To(BeTrue(), fmt.Sprintf("Create API call produced error %s. Error type wanted is %s.", reflect.TypeOf(err), reflect.TypeOf(&k8serrors.StatusError{})))
 			Expect(statusErr.Status().Message).Should(ContainSubstring("metadata.name max length is 63"))
 		})
 
 		It("should deny creating API with invalid name starting with non-alphanumeric character", func() {
-			var name = "-abcdef-123456789-123456789-123456789-123456789-123456789"
+			var nameStartingWithNonAlphanum = "-abcdef-123456789-123456789-123456789-123456789-123456789"
```

Comment:
please make it as a global variable, so that it can be shared between all the resources.